src/flutter/shell/platform/linux_embedded/surface/linuxes_surface_gl_drm_eglstream.h

```diff
@@ -0,0 +1,26 @@
+// Copyright 2021 Sony Corporation. All rights reserved.
```

Comment:
`linuxes_surface_gl_drm_gbm.h` and `linuxes_surface_gl_drm_eglstream.h` are a bit waste. Please remove these files by adding `SwapBuffer` into native_window_drm*.h

Reply:
I fixed it.